lib/icedfrisby.js

```diff
@@ -557,10 +556,7 @@ class Frisby {
           time: diff
         }
 
-        // call caller's callback
-        if (cb && typeof cb === "function") {
-          cb(self.current.response)
-        }
+        cb()
```

Comment:
This function invoked from `this._makeRequest()`, which of course has access to `this.current`, so there's no need for this check nor the parameter.

Reply:
having `cb()` without `return` before it always makes me nervous